p/pillow/build_info.json

```diff
@@ -2,22 +2,19 @@
   "maintainer": "Harithanagothu2",
   "package_name": "pillow",
   "github_url": "https://github.com/python-pillow/Pillow",
-  "version": "10.0.1",
+  "version": "11.0.0",
```

Comment:
Open-ce used pillow : 11.1.0

Reply:
As per discussion with @Harithanagothu2 .. it is understood that the build system picks up all versions in 11.* series for build 
Based on this approving the PR